core/deployment/src/main/java/io/quarkus/deployment/pkg/builditem/JarBuildItem.java

```diff
@@ -11,6 +10,8 @@
 import io.quarkus.deployment.pkg.PackageConfig;
 import io.quarkus.sbom.ApplicationManifestConfig;
 
+import static io.quarkus.deployment.pkg.PackageConfig.JarConfig.JarType.MUTABLE_JAR;
```

Comment:
Don't change the import order in an inconsistent way... (hint: it's static empty-line java empty-line everything else)

Reply:
finally added the plugin. Thanks